docs/source/install.rst

```diff
@@ -170,6 +170,16 @@ For the default build, running ``firedrake-configure`` with
 .. literalinclude:: petsc_configure_options.txt
    :language: text
 
+.. note::
+   If you install MPI through PETSc by passing ``--download-openmpi`` or
+   ``--download-mpich`` it is helpful to run the command::
+
+      $ export PATH=$PETSC_DIR/$PETSC_ARCH:$PATH
+
+
+   where ``PETSC_DIR=/path/to/petsc`` and ``PETSC_ARCH=arch-firedrake-default``.
+   This will allow the MPI executables (``mpicc``, ``mpiexec``, etc) to be found.
```

Comment:
```suggestion
   This will allow the MPI executables (``mpicc``, ``mpiexec``, etc) installed by
   PETSc to be found before any other versions installed on your machine.
```